Sources/BraintreeCore/ConfigurationLoader.swift

```diff
@@ -74,4 +82,11 @@ class ConfigurationLoader {
             }
         }
     }
+    
+    // MARK: - Private Methods
+    
+    func notifyCompletions(_ configuration: BTConfiguration?, _ error: Error?) {
+        pendingCompletions.forEach { $0(configuration, error) }
```

Comment:
This is a nit pick, you can disregard, but I think we are assuming this being called from main
as we do our async completion in the get call. Do we want to make those consistent?